CHANGELOG.md

```diff
@@ -9,15 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Added
 
+* Added `--cache-dir` CLI option for download caching ([#5](https://github.com/stjude-rust-labs/cloud-copy/pull/5)).
 * Added testing with emulators Azurite and localstack ([#4](https://github.com/stjude-rust-labs/cloud-copy/pull/4)).
 * Added initial implementation of `cloud-copy` ([#1](https://github.com/stjude-rust-labs/cloud-copy/pull/1)).
 
 #### Changed
 
+* Reverted the default parallelism back to the system's available parallelism ([#5](https://github.com/stjude-rust-labs/cloud-copy/pull/5)).
+* User agent now includes link to GitHub repository ([#5](https://github.com/stjude-rust-labs/cloud-copy/pull/5)).
 * Changed the default parallelism to 4 times the available parallelism ([#2](https://github.com/stjude-rust-labs/cloud-copy/pull/2)).
```

Comment:
Unless there were other effects that survive from #2, I don't think it's worth including both of these entries.

```suggestion
* User agent now includes link to GitHub repository ([#5](https://github.com/stjude-rust-labs/cloud-copy/pull/5)).
```

Though now that I look at the rest of the file, I see we don't actually have any releases yet. I am unsure whether it makes sense to have "Changed" or "Fixed" sections at all given that.

Reply:
Personally, I think it's fine to have those sections in an initial release as it can show the evolution of the software from the start.

Reply:
Also, we've basically been treating CHANGELOGs as effectively pull request history. We don't really curate them to the degree required to "negate" previous entries.